Services/WebContent/main.cpp

```diff
@@ -153,6 +156,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         Web::Fetch::Fetching::g_http_cache_enabled = true;
     }
 
+    if (!echo_server_port_string_view.is_empty()) {
+        if (auto maybe_echo_server_port = echo_server_port_string_view.to_number<u16>(); maybe_echo_server_port.has_value())
+            Web::Internals::g_echo_server_port = maybe_echo_server_port.value();
```

Comment:
This global would feel less icky if it was hidden behind a setter and defined as a file-scope static in the Internals.cpp file, but meh.

Reply:
Forgot to comment before resolving -- this is now a static behind a getter/setter in `Internals.cpp`